src/main/java/hudson/plugins/clover/CloverBuildAction.java

```diff
@@ -296,7 +298,7 @@ public int getPackages() {
     
     @Override  
     public Collection<? extends Action> getProjectActions() {  
```

Comment:
getProjectActions is an interface method of SimpleBuildStep.LastBuildAction:
http://javadoc.jenkins-ci.org/jenkins/tasks/SimpleBuildStep.LastBuildAction.html